pyproject.toml

```diff
@@ -1,18 +1,44 @@
-[tool.poetry]
+[project]
 name = "tortoise-orm"
 version = "0.24.1"
 description = "Easy async ORM for python, built with relations in mind"
-authors = ["Andrey Bondar <andrey@bondar.ru>", "Nickolas Grigoriadis <nagrigoriadis@gmail.com>", "long2ice <long2ice@gmail.com>"]
-license = "Apache-2.0"
+authors = [{name="Andrey Bondar", email="andrey@bondar.ru"}, {name="Nickolas Grigoriadis", email="nagrigoriadis@gmail.com"}, {name="long2ice", email="long2ice@gmail.com"}]
+license = {text="Apache-2.0"}
 readme = "README.rst"
+keywords = ["sql", "mysql", "postgres", "psql", "sqlite", "aiosqlite", "asyncpg", "relational", "database", "rdbms", "orm", "object mapper", "async", "asyncio", "aio", "psycopg"]
+include = ["CHANGELOG.rst", "LICENSE", "README.rst"]
+# classifieres is dynamic because poetry will create Python classifiers automatically with value in `tool.poetry` section
+dynamic = [ "classifiers" ]
+requires-python = ">=3.9"
+dependencies = [
+    "pypika-tortoise (>=0.5.0,<1.0.0); python_version < '4.0'",
+    "iso8601 (>=2.1.0,<3.0.0); python_version < '4.0'",
+    "aiosqlite (>=0.16.0,<1.0.0)",
+    "pytz",
+]
+
+[project.optional-dependencies]
+accel = [
+    "ciso8601; sys_platform != 'win32' and implementation_name == 'cpython'",
+    "uvloop; sys_platform != 'win32' and implementation_name == 'cpython'",
+    "orjson",
+]
+asyncpg = ["asyncpg"]
+aiomysql = ["aiomysql"]
+asyncmy = ["asyncmy (>=0.2.8,<1.0.0); python_version < '4.0'"]
+psycopg = ["psycopg[pool,binary] (>=3.0.12,<4.0.0)"]
+asyncodbc = ["asyncodbc (>=0.1.1,<1.0.0); python_version < '4.0'"]
+
+[project.urls]
 homepage = "https://github.com/tortoise/tortoise-orm"
 repository = "https://github.com/tortoise/tortoise-orm.git"
 documentation = "https://tortoise-orm.readthedocs.io"
-keywords = ["sql", "mysql", "postgres", "psql", "sqlite", "aiosqlite", "asyncpg", "relational", "database", "rdbms", "orm", "object mapper", "async", "asyncio", "aio", "psycopg"]
+
+[tool.poetry]
+requires-poetry = ">=2.0"
```

Comment:
We shouldn't be limiting the poetry version.

Reply:
Just tested this, you can still run `poetry add` but `poetry run` with the new format poetry < 2 fails with 
```
The Poetry configuration is invalid:
  - The fields ['authors', 'description', 'name', 'version'] are required in package mode.
```
So I guess we need to keep it.